include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -1456,15 +1456,15 @@ __remove_elements(_ExecutionPolicy&& __exec, _ForwardIterator __first, _ForwardI
             },
             ::std::plus<_DifferenceType>(),
             [=](_DifferenceType __i, _DifferenceType __len, _DifferenceType __initial) {
-                __internal::__brick_copy_by_mask(
-                    __first + __i, __first + __i + __len, __result + __initial, __mask + __i,
-                    [](_ForwardIterator __x, _Tp* __z) {
-                        if constexpr (::std::is_trivial_v<_Tp>)
-                            *__z = ::std::move(*__x);
-                        else
-                            ::new (::std::addressof(*__z)) _Tp(::std::move(*__x));
-                    },
-                    __is_vector);
+                __internal::__brick_copy_by_mask(__first + __i, __first + __i + __len, __result + __initial,
```

Comment:
What was changed in these lines?

Reply:
Seems only clang formatting for CI

Reply:
Quick plug for the clang format delta PR #715, which needs review and would have made the CI not suggest this change, away from the changed lines.

Reply:
I believe one main result of our from our discussion on clang-format was that "it is not the boss".  

Therefore, I believe we have the choice if it is worth "touching" these lines in the git history for this formatting change.  In my opinion, I think its probably OK to revert this section, and merge with a failed clang-format github runner (for this specific section).

To me, they are equivalently readable, so it my vote would be to delay reformatting it until the code is actually "touched" by a real change.

Reply:
Fixed : restored old formatting.